kolibri_explore_plugin/assets/src/modules/topicsTree/index.js

```diff
@@ -10,6 +10,8 @@ function defaultState() {
   return {
     channel: {},
     content: {},
+    contentId: null,
+    topicId: null,
```

Comment:
Can you explain in the commit message why are these new states needed? I think I get it, but it can be confusing as at first glance they may be derived from the state above. Ie. contentId can be content.id. Or if it's an ID for a different content, then we need a more descriptive name for the variable.

Reply:
I've updated this code to use a unique variable and I think now is clear. The idea was to be able to link to content and topic easily without messing around with the existing app, because for the custom app presentation, the content points to the channel node, so it's not possible to reuse that.